agent/engine/service_connect/manager_linux.go

```diff
@@ -119,9 +135,10 @@ func (m *manager) initAgentDirectoryMounts(taskId string, container *apicontaine
 
 func (m *manager) initAgentEnvironment(container *apicontainer.Container) {
 	scEnv := map[string]string{
-		m.relayENV:   filepath.Join(m.relayPathContainer, m.relayFileName),
+		m.relayENV:   unixRequestPrefix + filepath.Join(m.relayPathContainer, m.relayFileName),
```

Comment:
Should we update [defaultRelayPathContainer](https://github.com/aws/amazon-ecs-agent/blob/feature/service-connect/agent/engine/service_connect/manager_linux.go#L28) to **/var/run/ecs/** as an ENV for AppNet Relay as xDS endpoint is `APPMESH_XDS_ENDPOINT=unix:///var/run/ecs/envoy_xds.sock`, which does not contain **relay** in the path?

Reply:
Where are you getting that it should not have relay in it? The method that we use to mount the directories it has to be the full current defaultRelayPathContainer.

Reply:
Thanks for the clarification. Also confirm with [jwang186](https://github.com/jwang186) offline that the value is unix:///var/run/ecs/relay/envoy_xds.sock.

Reply:
Great thanks for the confirmation